src/lib/maxim.ts

```
@@ -509,6 +517,8 @@ export class Maxim {
 							versionId: deployedVersion!.id,
 							version: deployedVersion!.version,
 							nodes: deployedVersion!.config?.nodes.filter((n) => "prompt" in n),
+							run: (input: string, options?: { variables?: { [key: string]: string } }) =>
+								this.APIService.promptChain.runPromptChainVersion(deployedVersion!.id, input, options),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)